### ISSUE TITLE
Plot BCE loss after training

### ISSUE THREAD

**Issue by joepettinelli7** [joepettinelli7/llama_or_duck#5]
- Helpful to visualize training loss.
- Loss should be the loss from the validation data.
